### ISSUE TITLE
Login via CLI

### ISSUE THREAD

**Issue by hendrix04** [rmcrackan/Libation#1040]
**Is your feature request related to a problem? Please describe.**
I am a power user and prefer to do things in a headless way whenever possible. Currently, there is no way to login to Audible outside of the UI.

**Describe the solution you'd like**
A way to login via the command line. Ideally it would be fully self contained and wouldn't open a browser, but that would require setting up too much stuff on a user's computer and would probably cause more support tickets than it is worth.

**Describe alternatives you've considered**
Installing the application on another machine with a GUI to do the login process.
Alternatively, a secondary script could be used to handle the oauth flow and get the tokens outside of the application.

**Comment by rmcrackan** [comment]
> Installing the application on another machine with a GUI to do the login process.

This should generate a `AccountsSettings.json` file which you can just copy/paste in the correct dir.

**Comment by hendrix04** [comment]
Sure, that is a hacky work around though. I was hoping for something that didn't require me to copy files from one computer to another.

**Comment by rmcrackan** [comment]
Agreed. I'm unlikely to get to this anytime soon as a legit feature though so I at least wanted to give you that option.

**Comment by rmcrackan** [comment]
This (full cli features) is one of a number of far-reaching features that have been given a lot of thought for a long time with no commitments. I think it's fair to say we're not intending to do it in the near term.
